MANIFEST.in

```diff
@@ -56,6 +56,7 @@ include packages/examples/mqtt/aws_iot/simulated_home_auto_gen_topic.py
 include packages/examples/mqtt/aws_iot/simulated_home_topic_per_metric.py
 include packages/examples/mqtt/iotcc/iotcc_mqtt_edge_system_stats.py
 include packages/examples/mqtt/iotcc/iotcc_mqtt_ram.py
+include packages/examples/mqtt/iotcc/iotcc_resource_properties_mqtt.py
```

Comment:
In future, change these example file names to "organization_group_properties" instead of resource_properties. resource_properties doesn't sound correct.